python-threatexchange/threatexchange/extensions/vpdq/vpdq_util.py

```diff
@@ -11,6 +11,10 @@
 HASH = "hash"
 TIMESTAMP = "timestamp"
 VPDQ_TIMESTAMP_PRECISION = 3
+BITS_IN_VPDQ = 256
```

Comment:
(ignorable): I think you could use BITS_IN_PDQ since this is part of the threatexchange library